web/src/i18n/index.ts

```diff
@@ -32,7 +32,7 @@ export async function loadLocaleMessages(i18n: I18n, locale: Locale) {
 export default setup({
     legacy: false,
     locale: "en",
-    fallbackLocale: "en",
+    fallbackLocale: ['en'],
```

Comment:
We're using double quotes usually everywhere. Do `pnpm prettier` to automatically fix most things.